src/prompts/neuro.js

```diff
@@ -47,9 +46,45 @@ export default [
   'admits to murder',
   'rants',
   'new clone',
-  'chat goes HUH',
+  'chat spams HUH',
   'mentions favorite food',
   'wot? / what?',
   'claims to stream later',
-  'donowalled'
+  'donowalled',
+  'mentions turtles',
+  'chat spams AintNoWay',
+  'mentions random country',
+  'chat spams AYAYA',
+  'believe in the time god',
+  'KEKWa',
+  'Noted',
+  'chat spams Clueless',
+  'POG',
+  'PAUSE-sama',
+  'chat spams RIGGED',
+  'mentions John Lemon',
+  'D',
+  'chat spams RIPBOZO',
+  'mentions Abandoned Archive',
+  'SNIFFA',
+  'Classic',
+  'chat spams SoCute',
+  'mentions Bocchi The Rock',
+  'Technical Difficulties screen',
+  'Hype train lvl 5+',
+  'raiding anny or raided by anny',
+  'admits she\'s an AI',
+  'meaning of life',
+  'chat spams WAYTOODANK',
+  'Why?',
+  'world or humankind conquering plan',
+  'neuroWide',
+  'chat spams Vedal\'s catchphrase',
```

Comment:
```suggestion
  'chat spams Vedal\'s copypasta',
```

Or maybe
```suggestion
  'this is crazy 🐢',
```
We don't have much space for the sentence.